code/game/objects/items/devices/paicard.dm

```diff
@@ -14,6 +14,7 @@
 	///what emotion icon we have. handled in /mob/living/silicon/pai/Topic()
 	var/emotion_icon = "off"
 	resistance_flags = FIRE_PROOF | ACID_PROOF | INDESTRUCTIBLE
+	emag_toggleable = TRUE
```

Comment:
don't do this, override can_emag and return true like the sleeper does

Reply:
done